src/seedu/addressbook/parser/Parser.java

```diff
@@ -16,16 +31,14 @@
 
     public static final Pattern PERSON_INDEX_ARGS_FORMAT = Pattern.compile("(?<targetIndex>.+)");
 
-    public static final Pattern KEYWORDS_ARGS_FORMAT =
```

Comment:
Next time do the large scale style fixes in another branch or just do them directly on master. It's hard to review your feature when it's mixed in with style fixes